components/purchase_modal/purchase_modal.tsx

```diff
@@ -219,6 +417,38 @@ function Card(props: CardProps) {
             </div>
         </div>
     );
+
+    const monthlyYearlyPlan = (
+        <div className='PlanCard'>
+            <div className={containerClassname}>
```

Comment:
You can perhaps pass all className changes into a `getClassname` function for visibility. And if there is no className added to the container, you can pass `null` as to avoid snapshots adding the extra blank space.